core/tests/org.openjdk.jmc.jdp.test/src/test/java/org/openjdk/jmc/jdp/client/TestToolkit.java

```diff
@@ -148,4 +150,62 @@ public int getMaxHeartBeatTimeout() {
 		}
 		return bytes;
 	}
+
+	/**
+	 * See https://bugs.openjdk.java.net/browse/JMC-7539
+	 */
+	public static boolean areBroadcastingTestsEnabled() {
+		if (System.getProperty("skipJDPMulticastTests", "false").equals("true")) {
+			JDPClientTest.LOGGER.log(Level.INFO, "Broadcasting related tests are disabled");
+			return false;
+		}
+		if (!isBroadcastingPossible()) {
+			String os = System.getProperty("os.name").toLowerCase();
+			String helpMessage = ", you could try ";
+			String address = TEST_MULTICAST_ADDRESS.getCanonicalHostName();
+			if (os.startsWith("mac") || os.startsWith("darwin")) {
+				helpMessage += String.format("'sudo route add -host %s -interface en0'", address);
+			} else if (os.startsWith("linux")) {
+				helpMessage += String.format("'sudo ip route add %s dev eth0'", address);
+			} else {
+				helpMessage = "";
```

Comment:
On windows you can recommend to `route add ... <gateway>` in a shell with administrator permissions. I did that and this made the test work. Obviously the user has to look up the gateway.

Something like
`helpMessage += String.format("'route add %s <gateway>' in a shell with administrator permissions", address);`